app/controllers/admin_controller.rb

```diff
@@ -20,4 +20,16 @@ def show
   def require_admin
     raise CanCan::AccessDenied.new, 'Forbidden' unless current_user.has_role? :admin
   end
+
+  def sub_yml_setting(key, val)
```

Comment:
Not saying it's wrong, but how come `sub_yml_setting` got moved here from `SettingsController`?